src/breaking-change.ts

```diff
@@ -54,122 +91,138 @@ function blobHref(file: unknown) {
  * @param newSpec Path to the new swagger specification file.
  */
 async function runOad(oldSpec: string, newSpec: string) {
-  if (oldSpec === null || oldSpec === undefined || typeof oldSpec.valueOf() !== 'string' || !oldSpec.trim().length) {
-    throw new Error('oldSpec is a required parameter of type "string" and it cannot be an empty string.');
+  if (
+    oldSpec === null ||
+    oldSpec === undefined ||
+    typeof oldSpec.valueOf() !== "string" ||
+    !oldSpec.trim().length
```

Comment:
actually this is check could extract as a common function

Reply:
It's just format change. I will refactor code in the future.